static/js/services/recurring-processes-manager.js

```diff
@@ -0,0 +1,54 @@
+var UPDATE_RELATIVE_TIME = require('../modules/constants').UPDATE_RELATIVE_TIME;
+
+/*
+  Is responsible for managing and storing recurring processes.
+  Provides an interface for turning on a recurring process of predefined
+  type and turning it off when needed.
+ */
+angular.module('inboxServices').factory('RecurringProcessesManager',
+  function($interval, DomMutators) {
+    'use strict';
+    'ngInject';
+
+    var recurringProcesses = {};
+
+    function getProcessOptionsByName(name) {
+      switch (name) {
+        case UPDATE_RELATIVE_TIME:
+          return {
+            interval: 1000 * 60,
+            callback: DomMutators.updateRelativeTime
+          };
+        default:
+          throw new Error('Unknown recurring process name');
+      }
+    }
+
+    function startProcessByName(name, options) {
```

Comment:
I don't see the benefit of having named processes. Why not just have `startUpdateRelativeTime` and `stopUpdateRelativeTime` functions?